### PR TITLE
fix: modification that the icons are part of the class

### PR DESCRIPTION
**Pull request by GusBaamonde**
**Modification that the icons are part of the class**

iconos-parte-de-clase

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

**Review by lucianaDilizio — COMMENTED** [review]
Nice 😄

File: assets/scss/modules/_card.scss [hunk 1]
```diff
@@ -206,9 +206,12 @@
   /// icons card expanded
 
   .dp-icostar {
+    background: url('../img/ico-star.svg') 0 0 no-repeat;
```
Comment (cbernat) [inline assets/scss/modules/_card.scss:209]: This will need to be fixed in cards screen when this change applies to webapp.

Comment (cbernat) [reply assets/scss/modules/_card.scss:209]: I've talked this with Gus. We will fix in webapp.

**Pull request merged**